### PR TITLE
pager

### LINKED ISSUE #1: Support pager
Thank you for this tool!  Would you be willing to add an option for it to pipe output to less or another pager (IO::Pager?), or use a pager (Term::Pager?) internally?  Thanks for considering this request!

### PR DESCRIPTION
**Pull request by skaji**
Fix #1

### CODE REVIEW

**Comment by cxw42** [comment]
Thank you for making this change!  I am going to leave a line comment.

File: lib/App/tldr.pm [hunk 1]
```diff
@@ -31,12 +33,30 @@ sub parse_options {
...
+    my $self = shift;
+
+    my $cmd;
+    for my $try (grep $_, @_, $ENV{PAGER}, "less", "more") {
+        if (my $found = File::Which::which($try)) {
+            $cmd = $found, last;
+        }
+    }
+    return if !$cmd;
+    [$cmd, "-R"];
```
Comment (cxw42) [inline lib/App/tldr.pm]: `less` works fine for me, but not `more` -

    $ tldr --pager more tar
    more: unknown option -R
    Try 'more --help' for more information.

Perhaps `[$cmd, $cmd eq 'less' ? '-R' : ()]` ?

`tldr tar | more` works fine on my Cygwin x64 - `more` seems to pass the escape sequences unchanged, so colors appear properly.

File: lib/App/tldr.pm [hunk 2]
```diff
@@ -31,12 +33,30 @@ sub parse_options {
...
+    }
     push @{$self->{platform}}, $self->_guess_platform, "common";
     $self;
 }
 
+sub _guess_pager {
+    my $self = shift;
+
+    my $cmd;
+    for my $try (grep $_, @_, $ENV{PAGER}, "less", "more") {
```
Comment (cxw42) [inline lib/App/tldr.pm:51]: This changes the default behaviour to always use a pager.  That works fine for me!  But what if someone wants no-pager by default?  Perhaps remove `less` and `more` from the list, or add a configuration option?  I am trying to think ahead.

File: lib/App/tldr.pm (hunk 1 above)
Comment (skaji) [reply lib/App/tldr.pm]: Thanks. Done cdddbe5

File: lib/App/tldr.pm (hunk 2 above)
Comment (skaji) [reply lib/App/tldr.pm:51]: I think it is OK to use pager by default.

**Comment by cxw42** [comment]
This works for me with cdddbe5.  Thank you!

**Pull request merged**